src/tasks/predict_modality/methods/novel/run/config.vsh.yaml

```
@@ -0,0 +1,26 @@
+__merge__: ../../../api/comp_method_train.yaml
+functionality:
+  name: novel
+  info:
+    label: Novel
+    summary: A training method using encoder-decoder MLP model
+    description: This method trains an encoder-decoder MLP model with one output neuron per component in the target. As an input, the encoders use representations obtained from ATAC and GEX data via LSI transform and raw ADT data. The hyperparameters of the models were found via broad hyperparameter search using the Optuna framework.
+    documentation_url: https://github.com/openproblems-bio/neurips2021_multimodal_topmethods/tree/main/src/predict_modality/methods/novel#readme
+    repository_url: https://github.com/openproblems-bio/neurips2021_multimodal_topmethods/tree/main/src/predict_modality/methods/novel
+    reference: pmlr-v176-lance22a
```

this publication is already in the `library.bib` under `lance2022multimodal`.

```suggestion
    reference: lance2022multimodal
```